admin/CMakeLists.txt

```diff
@@ -34,3 +34,27 @@ file(WRITE "${LOCAL_INIT_DIR}/herbert_on.m" "${_herbert_on_script}")
 
 # Copy Herbert's worker template
 configure_file("${Herbert_WORKER_TEMPLATE}" "${LOCAL_INIT_DIR}/worker_v2.m")
+
+# =============================================================================
+# Install commands
+# =============================================================================
+# Function to install a file whose name ends in .template
+function(install_template_file)
+    set(_template_file "${ARGV0}")
+    get_filename_component(_template_base_name "${_template_file}" NAME)
```

Comment:
is this different from the Herbert variant to handle file files with paths?

Reply:
Out of necessity yes. In Herbert all the template files were in the `admin` directory and were passed into the function as relative paths. We now want to pass some absolute paths - which we need to trim down or the function will attempt to install the files to 
`C:\<CMAKE_INSTALL_PRFIX>\.\C:\absolute\path\to\template\file.template`.

Reply:
where does the absolute path come from?

Reply:
add comment to say this comes from find_herbert

Reply:
That's what the `Herbert_` prefix in the variable names being passed to the function is supposed to signify. It's common CMake convention - e.g. we know that all variables starting `Matlab_` come from the Matlab module

Reply:
here it's just `_template_file`  though with no clue where it's from.

Following back to where this is called you do have `Herbert_WORKER_TEMPLATE` so what is actually needed is a change to the `horace_FindHerbert.cmake` header to say this will be an absolute path.

Reply:
All found paths are absolute; there is nothing to clarify here.